packages/extension-base/src/hooks/useExtensionConfig.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useApplicationExtension} from '@salesforce/pwa-kit-extension-sdk/react'
+import Extension from '../setup-app'
```

Comment:
We talked about this in standup.. but I feel like this isn't the right place for that Id. 

1. Pulling in the entire class to use its static value might be overkill
2. It would be nice to have a shared place for the ID so that it can be used in the setup-server as well, it would be nice to have the symmetry

Maybe we create a constants.ts file and plop it in there? 

What do you think?

Reply:
discussed in slack, I added a new extension-meta.json at the root of the extension packages